src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -103,6 +103,10 @@ object Coroutine {
     var $yield: Y = null.asInstanceOf[Y]
     var $result: R = null.asInstanceOf[R]
 
+    /** Clones the coroutine that this instance is a part of.
```

Comment:
Sorry to insist here, but I'll be writing lint rules at some point to automate this, and I'd like to have the library in a clean state when I deploy them. We should have comments like this:

```
/** First line is a one-liner with a brief summary of the method.
 *
 *  After the empty line comes a potential longer description, separated from the asterisk with 2 spaces.
 *  Note here that the asterisk is indented with exactly 1 space, not 2.
 *
 *  After another empty line, comments come.
 */
```